_includes/workshops/card.html

```diff
@@ -14,14 +14,14 @@ <h2 class="h4"><a href="{{ workshop.url }}">{{ workshop.title }}</a></h2>
             {% if workshop.full == true %}
                 <p class="alert alert-danger">This workshop is full.</p>
             {% endif %}
-            <p>{{ workshop.content | truncatewords: 50 }} <a href="{{ workshop.url }}">more</a></p>
+            <p>{{ workshop.content | truncatewords: 50 }} <a href="{{ workshop.url }}"><span aria-label="more about {{ workshop.title }}">more</span></a></p>
         </div>
         <div class="col-12">
-            <h5>Time</h5>
-            <h5>{{ post_length }}</h5>
+            <h3 class="h5">Time</h3>
+            <h3 class="h5">{{ post.startTime }} to {{ post.endTime }}</h3>
```

Comment:
I don't think the time itself should be a heading since it should be under the 'time' heading above logically

Reply:
That sounds good, or it could be all in one tag like `<h3 class="h5">Time: {{ post.startTime }}</h3>`. Having two lines for so little text is part of what stretches the boxes so much.